src/site/content/en/blog/compat2021-holiday-update/index.md

```diff
@@ -0,0 +1,150 @@
+---
+title: "Compat 2021 Holiday Update: presents for developers before the end of the year"
+subhead:
+    "End of the year update on Compat 2021—an effort to eliminate browser compatibility problems in five key focus areas: CSS Flexbox, CSS Grid, position: sticky, aspect-ratio, and CSS transforms."
+authors:
+  - foolip
+  - kosamari 
+hero: "image/Wd2sVnt4VLho4jgp7UtIyWFceE02/T4gzpGXXQ9sud6CEbHno.jpg" alt: "a hand holding a small box of presents with red ribbon." 
+date: 2021-12-10 
+updated: 2021-12-10 
+tags:
+  - blog
+  - css
+---
+
+The end of the year is near, and it's time for a final update on Compat 2021—an effort to eliminate 
+browser compatibility problems in five key focus areas.
+
+{% Aside %}
+This effort is also known as Interop 2021, matching the name of 
+[Interop 2022](https://github.com/web-platform-tests/interop-2022), which is the continuation of 
+this effort.
+{% endAside %}
+
+<div class="w-stats">
+ <div class="w-stat">
+   <p class="w-stat__figure">>90<sub>%</sub></p>
+   <p class="w-stat__desc">score in all browsers</p>
+ </div>
+</div>
+
+Since our [last update](https://web.dev/compat2021-midyear/), we’ve continued to see improvements 
+in all browsers. All browsers started with much lower test scores at the beginning of the year, but 
+now all browsers have surpassed 90%! This means the web platform has significantly improved 
+interoperability of 5 focus areas. 
+
+<figure class="w-figure">
+{% Img src="image/Wd2sVnt4VLho4jgp7UtIyWFceE02/Kaz3ye7gFfow8ia7lWYW.png", alt="A snapshot of Compat 
+2021 Dashboard (experimental browsers)", width="800", height="727" %}
+  <figcaption class="w-figcaption">
+    A snapshot of <a href="https://wpt.fyi/compat2021">Compat 2021 Dashboard</a> (experimental 
+    browsers).
+  </figcaption>
+</figure>
+
+{% Aside 'caution' %}
+The graph does not correctly show the improvements in Safari Technology Preview between May and 
+November, as the version tested was [stuck](https://github.com/web-platform-tests/wpt/issues/31147) 
+at an older version. This is now resolved but the data between May and November remains incorrect.
+{% endAside %}
+
+Contributions to browser engines are made not only by browser vendors, but also others in the web 
+community. For this project, we particularly want to thank Igalia for 
+[their involvement](https://www.igalia.com/2021/11/12/New-Interoperability-Milestones.html) and 
+their continued work to improve the scores. Igalia has contributed to improving all five focus 
+areas of Compat 2021.
+
+On [wpt.fyi](https://wpt.fyi/), the test results dashboard, there’s now a 
+[filtered test results view](https://wpt.fyi/results/?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&q=label%3Ainterop-2021)
+([permalink](https://wpt.fyi/results/?q=label%3Ainterop-2021&run_id=5629744973348864&run_id=5117500464300032&run_id=5734588178497536)) showing all of the tests included in Compat 2021, 
```

Comment:
from the context it's not totally clear to me that this means a permalink of both the test filtering _and_ the specific runs; maybe be explicit, or just expand it to "(as of today)" or something?

Reply:
I'll remove it. The 3 runs that were current can be dug out of the 3 following links, and I'm pretty sure I'm the only person who ever might.